source/includes/inkling-reference/_concepts.html.md

```diff
@@ -59,6 +59,25 @@ The concept statement specifies input sources and output targets. Input sources
 are listed after the `follows` keyword and output sources are listed after the
 `feeds` keyword. 
 
+```inkling--code
+concept bar is classifier
+  predicts (Move)
+  # feeds foo            not necessary 
```

Comment:
Should this be

`feeds foo             # not necessary`

or are you trying to show that this would be commented out?

Reply:
the latter: not necessary, so commented out